### PR TITLE
Update title sequences to v0.4.14 release

### PR DESCRIPTION
**Pull request by Broxzier**
Depends on https://github.com/OpenRCT2/title-sequences/pull/13
Depends on https://github.com/OpenRCT2/title-sequences/pull/14

Also create a release (tag) and add title-sequences.zip to the release, so that the links work. The sha1 hash should be correct.

### CODE REVIEW

File: distribution/changelog.txt [hunk 1]
```diff
@@ -8,6 +8,7 @@
 - Feature: [#22213] [Plugin] Allow plugins to focus on textboxes in custom windows.
 - Feature: [#22272] [Plugin] Expose ride vehicle’s current track type via car trackLocation.
 - Feature: [#22301] Loading save games or scenarios now indicates loading progress.
+- Feature: [#22472] New title sequences (https://github.com/OpenRCT2/title-sequences/releases/tag/v0.4.13).
```
Comment (AaronVanGeffen) [inline distribution/changelog.txt:11]: Having a URL in the changelog like this is a bit odd, I think. Maybe we could mention the PR# on the title sequence repo instead?

Comment (Gymnasiast) [reply distribution/changelog.txt:11]: Or just leave it out - it already points to this PR after all, where people can click through if they want to know more.

Comment (Broxzier) [reply distribution/changelog.txt:11]: Hmm, last year you asked for a way to credit the creators more easily, and linking to the release page where their names will be listed was whta I chose to do. Linking to the PR or to this one is still at least one more click away form their credits.
https://github.com/OpenRCT2/OpenRCT2/pull/20680#issuecomment-1664254930

Comment (MaxMallon) [reply distribution/changelog.txt:11]: Maybe put the names in the credits page?

Comment (Gymnasiast) [reply distribution/changelog.txt:11]: We could also put their names into contributors.md.

Nevertheless, if we must do it this way, we should at least clarify why the URL is there.

```suggestion
- Feature: [#22472] New title sequences (see https://github.com/OpenRCT2/title-sequences/releases/tag/v0.4.13 for credits).
```

**Comment by Broxzier** [comment]
Depends on https://github.com/OpenRCT2/title-sequences/pull/14

File: openrct2.proj [hunk 2]
```diff
@@ -42,7 +42,7 @@
     <LibsUrl Condition="'$(Platform)'=='x64'">https://github.com/OpenRCT2/Dependencies/releases/download/v36/openrct2-libs-v36-x64-windows-static.zip</LibsUrl>
     <LibsSha1 Condition="'$(Platform)'=='x64'">3816671560af3571ada728ffeceec0443471c03b</LibsSha1>
     <LibsUrl Condition="'$(Platform)'=='Win32'">https://github.com/OpenRCT2/Dependencies/releases/download/v36/openrct2-libs-v36-x86-windows-static.zip</LibsUrl>
-    <LibsSha1 Condition="'$(Platform)'=='Win32'">d2b8d98a9711cf319e3bd5330e77aef5395820f1</LibsSha1>
```
Comment (Gymnasiast) [inline openrct2.proj:45]: Looks like you changed the wrong SHA sum here.

File: openrct2.proj [hunk 3]
```diff
@@ -42,7 +42,7 @@
...
+    <LibsSha1 Condition="'$(Platform)'=='Win32'">6c04781b959b468e1f65ec2d2f21f5aaa5e5724d</LibsSha1>
     <TitleSequencesUrl>https://github.com/OpenRCT2/title-sequences/releases/download/v0.4.14/title-sequences.zip</TitleSequencesUrl>
     <TitleSequencesSha1>be28b18a250176036e860fcf5cc85ab6c200fd54</TitleSequencesSha1>
```
Comment (Gymnasiast) [inline openrct2.proj:47]: This looks like the SHA to modify.

**Review by Gymnasiast — APPROVED** [review]

**Review by AaronVanGeffen — APPROVED** [review]

**Pull request merged**